src/utils/format.ts

```diff
@@ -207,3 +207,9 @@ export function capitalize(sentence: string): string {
     .map((word) => word[0].toUpperCase() + word.slice(1).toLowerCase())
     .join(' ')
 }
+
+export function getSmallLimit(decimals?: number): string {
```

Comment:
It took me a bit to get what it was. 

Maybe another name, in the direction of `getMinimumRepresentableValue`